version.txt

```diff
@@ -1 +1 @@
-0.10.0a0
+0.11.0a0
```

Comment:
The last release was 0.9 so the one happening now is 0.10. Is the +1 offset expected here?

Reply:
Good question. Yes our master is always one version ahead. I'll write a guide for it shortly so that we are all aware. :)

Reply:
Yes, we are releasing 0.10, so nightlies should be at 0.11 so that we don't have version issues that the nightlies are "older" than the latest released version